[2025-04-17T23:30:03.399+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_loader.create_tables scheduled__2025-04-17T23:25:00+00:00 [queued]>
[2025-04-17T23:30:03.406+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_loader.create_tables scheduled__2025-04-17T23:25:00+00:00 [queued]>
[2025-04-17T23:30:03.406+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-17T23:30:03.416+0000] {taskinstance.py:1382} INFO - Executing <Task(SQLExecuteQueryOperator): create_tables> on 2025-04-17 23:25:00+00:00
[2025-04-17T23:30:03.419+0000] {standard_task_runner.py:57} INFO - Started process 324 to run task
[2025-04-17T23:30:03.421+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'parquet_loader', 'create_tables', 'scheduled__2025-04-17T23:25:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_postgres_dbt.py', '--cfg-path', '/tmp/tmpm6rrwwv4']
[2025-04-17T23:30:03.423+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask create_tables
[2025-04-17T23:30:03.473+0000] {task_command.py:416} INFO - Running <TaskInstance: parquet_loader.create_tables scheduled__2025-04-17T23:25:00+00:00 [running]> on host fe2f3e6bd690
[2025-04-17T23:30:03.547+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parquet_loader' AIRFLOW_CTX_TASK_ID='create_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T23:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-17T23:25:00+00:00'
[2025-04-17T23:30:03.548+0000] {sql.py:274} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS staging_table (
            id INT,
            name TEXT,
            age INT
        );
        
        CREATE TABLE IF NOT EXISTS main_table (
            id INT PRIMARY KEY,
            name TEXT,
            age INT
        );
        
[2025-04-17T23:30:03.558+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:30:03.570+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:30:03.576+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 385, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 155, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"

[2025-04-17T23:30:03.586+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=parquet_loader, task_id=create_tables, execution_date=20250417T232500, start_date=20250417T233003, end_date=20250417T233003
[2025-04-17T23:30:03.596+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 25 for task create_tables (connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"
; 324)
[2025-04-17T23:30:03.601+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-04-17T23:30:03.624+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
